Nodejs/Product/Nodejs/ProjectTemplates/Express4App/ExpressApp.vstemplate

```diff
@@ -12,6 +12,11 @@
     <NumberOfParentCategoriesToRollUp>1</NumberOfParentCategoriesToRollUp>
     <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
     <PreviewImage>Preview.png</PreviewImage>
+    <LanguageTag>Javascript</LanguageTag>
+    <PlatformTag>Windows</PlatformTag>
+    <PlatformTag>Linux</PlatformTag>
+    <PlatformTag>macOS</PlatformTag>
```

Comment:
I would have guessed "MacOS", but I'm assuming you copied this from somewhere.

Reply:
It's the what autocomplete shows. The [documentation](https://docs.microsoft.com/en-us/visualstudio/ide/template-tags?view=vs-2019#built-in-tags) shows this as all lowercase but I don't think it matters as it is displaying the tags correctly .